tests/conftest.py

```diff
@@ -3,8 +3,22 @@
 from typing import Any
 from unittest.mock import MagicMock, patch
 
+from django.conf import settings
+
 import pytest
 from data_platform_catalogue.client.datahub_client import DataHubCatalogueClient
+from data_platform_catalogue.entities import (
```

Comment:
Why are there 2 import statements for `data_platform_catalogue.entities`? Also, can these imports be alphabetical (available in the command palette in VSCode).

Reply:
The result of rebasing main which I didn't spot. Now removed